packages/commonwealth/client/scripts/navigation/helpers.tsx

```diff
@@ -58,12 +58,17 @@ export const useCommonNavigate = () => {
    *  To navigate without prefix whatsoever, the null has to be passed.
    *  navigate("/privacy", {}, null)
    */
-  return (url: To, options?: NavigateOptions, prefix?: null | string) => {
+  return (
+    url: To,
+    options?: NavigateOptions,
+    prefix?: null | string,
+    communityPrefix = true
```

Comment:
Usually the workflow for navigation is:
1. pass it some relative path, ex /discussions
2. navigates to baseUrl + /[community]

For example, with dydx it would be:
commonwealth.im/dydx/discussions

The issue is, sometimes we don't want to navigate internally, but outside the community, like the /createCommunity example above. Without this option we would get:

1. Pass it /createCommunity
2. navigates to commonwealth.im/dydx/createCommunity

So we need an option to let the navigation know when we want to navigate internally but outside the community.